### PR TITLE
sonar fix.

### PR DESCRIPTION
**Pull request by Argalep**

### CODE REVIEW

File: Scripts/Abilities/MysticArc.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public override void OnHit(Mobile attacker, Mobile defender, int damage)
 
             if (m_Target != null)
             {
-                defender.MovingEffect(m_Target, weapon.ItemID, 18, 1, false, false);
+                defender?.MovingEffect(m_Target, weapon.ItemID, 18, 1, false, false);
```
Comment (TrueUO) [inline Scripts/Abilities/MysticArc.cs]: Why the ? 

Do we need that there?

Comment (Argalep) [reply Scripts/Abilities/MysticArc.cs]: image

Comment (TrueUO) [reply Scripts/Abilities/MysticArc.cs]: Is that a style change we want to use? We do not use the ? anywhere else to != null. I say we do all or none with the style changes.

Comment (kevin-10) [reply Scripts/Abilities/MysticArc.cs]: According to this: https://stackoverflow.com/questions/40992457/c-sharp-6-0-multiple-identical-null-conditional-operator-checks-vs-single-tradit

There is very little to no performance difference. Also stated that using the traditional null checks keep the code blocks nice and organized.

Comment (kevin-10) [reply Scripts/Abilities/MysticArc.cs]: It's not really a style, just a shortcut operator. It's like using ?? or the short conditional ?.

Comment (TrueUO) [reply Scripts/Abilities/MysticArc.cs]: I use a lot of Reshapers suggestions BUT I have noticed it sometimes is not right or relevant to our project. I love Reshaper but some stuff I do ignore.

**Review by TrueUO — APPROVED** [review]

**Review by kevin-10 — APPROVED** [review]

**Pull request merged**